portfolio.html

```diff
@@ -0,0 +1,603 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>portfolio</title>
+    <link rel="stylesheet" href="portfolio.css" />
+  </head>
+  <body>
+    <header class="nav-container">
+      <nav>
+        <div class="lower-bar">
+          <div class="haftamu">Haftamu</div>
```

Comment:
### Your Menu Icon
![menu-icon (copy)](https://user-images.githubusercontent.com/93039359/179871394-c4bba66a-ceb2-4d18-8723-8d1da14e06ad.png)
- [ ] You have made an effort but your menu icon is still visible in your design. Kindly use display: none; in your CSS
 file to hide it in the desktop version :blush: